docs/product-user-guide/src/main/asciidoc/case-management-roles-con.adoc

```diff
@@ -1,8 +1,39 @@
 [id='case-management-roles-con-{context}']
-= Case Roles
+= Case roles
 
-Case roles provide an additional layer of abstraction for user participation in case handling. Roles drive the authorization for the case instance, and can be used for user activities assignment. The only role that is predefined is `Owner`. A user, a group, or groups can be assigned to the owner role. The owner is whoever the case belongs to. This can be changed. Roles are not restricted to a single set of people or groups as part of case definition. 
+Case roles provide an additional layer of abstraction for user participation in case handling. Roles, users, and groups are used for different purposes in case management: 
 
-Roles are optional, and not a case requirement, however, it is considered to be best practice to use roles within case definitions.
+Role::
+Roles drive the authorization for the case instance, and can be used for user activities assignment. The only role that is predefined is `Owner`. A user, a group, or groups can be assigned to the owner role. The owner is whoever the case belongs to. Roles are not restricted to a single set of people or groups as part of case definition.  Roles should always be used for task assignment rather than assigning a specific user or group in order to ensure the case remains dynamic.
```

Comment:
Should it be: Roles drive the authorization for the case instance, and can be used for user activity assignments ?